content/index.js

```diff
@@ -245,11 +248,7 @@ cli
     return runMakePopularitiesFile(args.csvfile, options, logger);
   });
 
-cli.parse(process.argv).then((r) => {
-  // If the command explicitly returned a number, use that as the exit code
-  // Otherwise, if it's anything truthy return 1 or all else 0.
-  process.exitCode = typeof r === Number ? r : r ? 1 : 0;
-});
+cli.parse(process.argv);
```

Comment:
I think this works. But I'm not sure. At least it seems to work when I'm running `node content build ...` in various ways. I might have broken the dumper or something. 
I *could* spend some serious time investigating how you should properly use `carporal` with async and error handling etc. But there's still a chance we might go back to 100% sync which will simplify this stuff.